core/src/blocks/chat.rs

```diff
@@ -292,6 +292,11 @@ impl Block for Chat {
                     extras["response_format"] = json!(s.clone());
                 }
 
+                extras["use_tools"] = json!(v.get("use_tools").map_or(false, |v| match v {
```

Comment:
`map_or` does not seem useful here, can we simply match against `json!(...)` ?

Reply:
Fixed.